frontend/benefit/applicant/src/components/applications/forms/application/step5/attachmentsListView/AttachmentsListView.tsx

```diff
@@ -24,29 +24,25 @@ const AttachmentsListView: React.FC<AttachmentsListViewProps> = ({
     [attachments, type]
   );
 
-  return (
```

Comment:
There is a lot of code changes where unnecessary `<>` is dropped because eslint nags about it.